### PR TITLE
Bug/240 make form code optional

### PR DESCRIPTION
**Pull request by towa**
With this PR the injection of code into the layer forms is made optional.
A Checkbox is added to the ImportGMLAS Panel, the default is to not inject code in forms.

In reference to issue #240

### CODE REVIEW

**Comment by Guts** [comment]
Thanks @towa, I let @benoitblanc have a look to this part.

@towa are you working on your own or for BRGM or...?

**Comment by towa** [comment]
Thanks,
I work for @gbd-consult

File: gml_application_schema_toolbox/core/load_gmlas_in_qgis.py [hunk 1]
```diff
@@ -99,7 +99,9 @@ def createLayerTreeModelLegendNodes(self, layer_tree_layer):
         return [QgsSimpleLegendNode(layer_tree_layer, self.text, self.icon, self)]
 
 
-def import_in_qgis(gmlas_uri: str, provider: str, schema: Union[str, None] = None):
+def import_in_qgis(
+    gmlas_uri: str, provider: str, add_form_code: bool, schema: Union[str, None] = None
```
Comment (benoitblanc) [inline gml_application_schema_toolbox/core/load_gmlas_in_qgis.py:103]: Can you please provide a description of `add_form_code` parameter in docstring ?

**Review by benoitblanc — COMMENTED** [review]
Hi @towa , sorry for the delay. 

As I do not know if custom code is still relevant, I guess make it optional is a good way to do it.

Just one remark on docstring.

Thanks !

**Pull request merged**